docs/datamodel/computeds.rst

```diff
@@ -68,7 +68,8 @@ next time you try to :ref:`create a migration <ref_intro_migrations>`.
 
   type Person {
     property first_name -> str;
-    required property name_upper := str_upper(.name); # invalid
+    # invalid, because first_name is not a required property
```

Comment:
Could we add something to indicate what we're talking about? Maybe the pointing down emoji (👇 )? Not sure if that would be appropriate in this context, but I think it would make it less ambiguous.